### ISSUE TITLE
DCGAN for fashion-MNIST results inaccurate

### ISSUE THREAD

**Issue by banishnarang** [moelgendy/deep_learning_for_vision_systems#11]
When the DCGAN Network is trained for 1000 or even 10,000 epochs, the image results are just noise which is kind of consistent in all the images. Typed every line of code at first, tried running the file from the repo afterward, however, the results tend to be the same.

**Comment by moelgendy** [comment]
Thanks for the note, @banishnarang! I'll look into this issue and update the project as needed.

**Comment by AntonVeynshter** [comment]
I run in same issue. I tried to decrease learning rate for discriminator, it worked, but after some hundert epochs it became noise again.
